catalog/global-pasture-watch/projects_global-pasture-watch_assets_ggpp-30m_v1_ugpp_m.jsonnet

```diff
@@ -0,0 +1,111 @@
+local id = 'projects/global-pasture-watch/assets/ggpp-30m/v1/ugpp_m';
+local subdir = 'global-pasture-watch';
+local version = '1';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+local ugpp30m_v1 = importstr 'ugpp-30m_v1.md';
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  title: 'GPW Annual uncalibrated Gross Primary Productivity (uGPP) v' + version,
+  version: version,
+  description: |||
+    This dataset provides global uncalibrated EO-based Gross Primary Productivity 
+    from 2000 to 2024 at 30-m spatial resolution. 
```

Comment:
Do you plan to produce more years later? If so, better to not mention the end year here, as we might forget to update it later.